### ISSUE TITLE
执行python example_samples.py --dataset PUGAN --R 4 --step 30 --example_file ./example/KITTI.xyz报错

### ISSUE THREAD

**Issue by Philcalab** [QWTforGithub/PUDM#9]
你好，非常感谢您之前的邮件回复。下面是我的一些疑问：
1.我不太清楚您所说的pointnet2_ops前置包有哪些，我是执行了pip install pointnet2，然后看了下pointnet2的github（https://github.com/erikwijmans/Pointnet2_PyTorch#egg=pointnet2_ops&subdirectory=pointnet2_ops_lib），按照它的需求安装了一些包。之后有执行sh compile.sh,并未显示报错。

2.我的cuda是安装好的，nvcc -V的结果cuda-11.8。另外，我的机器好像无法直接通过 conda install cudatoolkit==11.1安装cuda-11.1，如果不加版本号，默认安装cudatoolkit==11.8。我也很奇怪为什么会报找不到cuda的错误，但是看报错好像找到了我本地安装的cuda-11.8。

3.按照您之前的回复，我尝试运行如下代码检查cuda相关信息
import torch
print(torch.__version__)
print(torch.cuda.is_available())
print(torch.cuda.device_count())
print(torch.backends.cudnn.version())
print(torch.version.cuda)

控制台输出：
/home/user/anaconda3/envs/pudm/bin/python /data/lc/PUDM/check_cuda.py 
1.9.0+cu111
/home/user/anaconda3/envs/pudm/lib/python3.9/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: CUDA driver initialization failed, you might not have a CUDA gpu. (Triggered internally at  /pytorch/c10/cuda/CUDAFunctions.cpp:115.)
  return torch._C._cuda_getDeviceCount() > 0
False
0
8005
11.1

我现在也不清楚为什么cuda没有被识别到，请问我是否可以本地安装cuda-11.8，conda环境 安装cudatoolkit==11.8，并且安装相应版本的torch呢？

**Comment by QWTforGithub** [comment]
你好，我认为问题出在cuda11.8上，建议降低本地的cuda版本到11.6或者更低。然后重新在虚拟环境中安装cudatookit和cudnn:
`conda install cudatoolkit
pip install nvidia-cudnn-cu11`
或者尝试：
`conda install cudatoolkit=11.1 -c nvidia/label/cuda-11.1.1 -c conda-forge
 conda install cudnn=8.2.1 -c nvidia/label/cudnn8.2.1`
再次验证：
`import torch
print(torch.version)
print(torch.cuda.is_available())
print(torch.cuda.device_count())
print(torch.backends.cudnn.version())
print(torch.version.cuda)`
